iFixit/react-commerce
# Upgrade Strapi to v4.6.1

closes #1349 

❗❗❗After this PR is merged Strapi should be redeployed in production ❗❗❗

cc @sterlinghirsh @danielbeardsley 

## QA

1. Login into the [strapi instance for this branch](https://upgrade-strapi-to-v4-6-0.govinor.com/admin)
2. Go to settings and check that the Strapi version is `v4.6.1`
3. Open [Vercel preview](https://react-commerce-git-upgrade-strapi-to-v4-6-0-ifixit.vercel.app/Parts)
4. Verify that product list page renders
